website/docs/learning-and-resources/sample-apps.md

```diff
@@ -582,7 +582,7 @@ import TabItem from '@theme/TabItem';
     </div>
 </div>
 
-### UI Components
+### UI components
```

Comment:
Please add this as an exception to Microsoft>> HeadingAcronyms. Let me know if you need help.

Reply:
added